commands/deploy.js

```diff
@@ -359,17 +360,15 @@ function logDetails(details, opts) {
       .value();
   }
 
-  if(details.runTestResult) {
+  if(details.runTestResult.any) {
     if(details.runTestResult.numFailures) {
       logger.error('test results ====>');
     } else {
       logger.success('test results ====>');
+      logger.list('tests run: ' + details.runTestResult.numTestsRun);
```

Comment:
Is there a reason why this is only logged out when the test run is a success now?